cypress.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
we already have this

Reply:
but we are not able to cypress on local without global cypress installed

Reply:
You can right? if you go to mobile-app folder?

Reply:
The idea is once desktop app folder is there, you need to run it there as well

Reply:
ops yep you're right, will revert this